src/Aspire.ProjectTemplates/templates/aspire-starter/AspireStarterApplication.1.sln

```diff
@@ -8,7 +8,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AspireStarterApplication.1.
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AspireStarterApplication.1.ApiService", "AspireStarterApplication.1.ApiService\AspireStarterApplication.1.ApiService.csproj", "{9FEB877E-015D-4E20-AE63-06C596E242E4}"
 EndProject
-#if (CreateTestsProject)
+#if (TestFramework != 'None')
```

Comment:
Good catch! I assume you verified this fixes it.

Reply:
@radical do we have test coverage the starter template creation? Can we add this as a case?

Reply:
Yep. I assume the conditionals may be case-insensitive, but matched the literal default value in template config and verified 'None' and 'none' both work

Reply:
What's the case that was caught here? Using `none` type as the framework generated a test project when it shouldn't have? I check for non-existence of a test project in case of `none` but I didn't see it failing.

Reply:
In the current shipping templates the option is CreateTestsProject since there was only one type: xunit.  Now that we added unit test framework choice, the option changed to TestFramework and is either none (no arg provided so no tests projects), xunit.net, nunit, or mstest